data/explorations/root_linear_coefficient_theorem.yaml

```diff
@@ -13,11 +13,11 @@ states:
     content:
       audio_translations: {}
       html: "<span>Here's that expression again:</span><div><oppia-noninteractive-math\
-        \ raw_latex-with-value=\"&amp;quot;(x - a_1)(x - a_2)(x - a_3)...(x - a_n)&amp;quot;\"\
+        \ math_content-with-value=\"{&amp;quot;raw_latex&amp;quot;: &amp;quot;(x - a_1)(x - a_2)(x - a_3)...(x - a_n)&amp;quot;,&amp;quot;svg_filename&amp;quot;:&amp;quot;&amp;quot;}\"\
```

Comment:
I'm not reviewing the code here, make sure to validate this manually by loading demo exploration from the admin page!

Reply:
Yes I have tested this manually